resources/application-connector/values.yaml

```diff
@@ -42,31 +42,31 @@ global:
       version: "PR-13670"
     application_connectivity_certs_setup_job:
       name: "application-connectivity-certs-setup-job"
-      version: "PR-13170"
+      version: "708f6a87"
     application_connectivity_validator:
       name: "application-connectivity-validator"
-      version: "PR-13170"
+      version: "cc89c542"
     application_gateway:
       name: "application-gateway"
-      version: "PR-13171"
+      version: "6d430445"
     application_operator:
       name: "application-operator"
-      version: "PR-12257"
+      version: "3f163e8f"
     application_registry:
       name: "application-registry"
-      version: "PR-12257"
+      version: "a0a4ec40"
     central_application_connectivity_validator:
       name: "central-application-connectivity-validator"
-      version: "PR-12963"
+      version: "708f6a87"
     central_application_gateway:
       name: "central-application-gateway"
-      version: "PR-13171"
+      version: "708f6a87"
     connection_token_handler:
       name: "connection-token-handler"
       version: "PR-12963"
     connector_service:
       name: "connector-service"
-      version: "PR-13221"
+      version: "a0a4ec40"
```

Comment:
PR `PR-13221` is merged as a commit [ae096c4](https://github.com/kyma-project/kyma/commit/ae096c4a18d06b200a0297e0b68f495ef9f28c26). Therefore, the bumped image should be `ae096c4a`

Reply:
For this we include additionally:

- #13643 Add the definition for Application to the Glossary
- #13379  support additional linters per component

Both of them are safe to include in this PR